awx/main/tests/functional/api/test_credential_input_sources.py

```
@@ -316,3 +319,60 @@ def test_create_credential_input_source_with_already_used_input_returns_400(post
+    # Mock the credential plugin backend
+    with mock.patch.object(external_credential.credential_type.plugin, 'backend') as mock_backend:
+        mock_backend.return_value = 'test_value'
+
+        # Call with context containing workload_identity_token
+        test_context = {'workload_identity_token': 'jwt_token_here'}
+        result = input_source.get_input_value(context=test_context)
+
+        # Verify backend was called with workload_identity_token
+        assert result == 'test_value'
+        call_kwargs = mock_backend.call_args[1]
+        assert call_kwargs['workload_identity_token'] == 'jwt_token_here'
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "workload_identity_token"

(S105)